runtime/CMakeLists.txt

```diff
@@ -365,6 +365,7 @@ if(J9VM_MODULE_J9VM)
 	add_subdirectory(j9vm)
 	add_subdirectory(j9vm_b150)
 	add_subdirectory(j9vm_b156)
+	add_subdirectory(j9vm_jdk11)
```

Comment:
Shouldn't there also be a reference to `j9vm_jdk12` here?

Reply:
fixed